### ISSUE TITLE
File.find fails on windows systems due to missing import Glob

### ISSUE THREAD

**Issue by cedricjacquet** [saltstack/salt#33331]
### Description of Issue/Question

If you try to find some files on windows system with file.find module we have an issue "global name glob is not defined"
### Steps to Reproduce Issue

```
ERROR file.find on Host (vmas018) doesn't returns a list of files : '
    The minion function caused an exception: Traceback (most recent call last):
    File "C:\\salt\\bin\\lib\\site-packages\\salt\\minion.py", line 1071, in _thread_return
    return_data = func(*args, **kwargs)
    File "C:\\salt\\bin\\lib\\site-packages\\salt\\modules\\file.py", line 694, in find
    ret = [item for i in [finder.find(p) for p in glob.glob(os.path.expanduser(path))] for item in i]\nNameError: global name \'glob\' is not defined
    '
```

after adding import in win_file everything worked again
### Versions Report

2015.8.8

**Comment by Ch3LL** [comment]
@cedricjacquet i was able to replicate this as well:

```
➜  ~ sudo salt 'win88' file.find C:/tmp name=test.txt
win88:
    The minion function caused an exception: Traceback (most recent call last):
      File "c:\salt\bin\lib\site-packages\salt\minion.py", line 1071, in _thread_return
        return_data = func(*args, **kwargs)
      File "c:\salt\bin\lib\site-packages\salt\modules\file.py", line 694, in find
        ret = [item for i in [finder.find(p) for p in glob.glob(os.path.expanduser(path))] for item in i]
    NameError: global name 'glob' is not defined
```

add the following to modules/win_file.py
`import glob`

```
➜  ~ sudo salt 'win88' file.find C:/tmp name=test.txt
win88:
    - C:/tmp\test.txt
```

@twangboy is this the right solution to just import glob?

**Comment by twangboy** [comment]
@Ch3LL Yes

**Comment by rallytime** [comment]
This was fixed with PR #35525, which is available in the 2016.3.3 release of Salt. Closing.
